src/getRealPath.js

```diff
@@ -43,40 +46,21 @@ function getRealPathFromRootConfig(sourcePath, currentFile, opts) {
   )));
 }
 
-function getRealPathFromAliasConfig(sourcePath, currentFile, { alias, cwd }) {
-  const moduleSplit = sourcePath.split('/');
-  let aliasPath;
-
-  while (moduleSplit.length) {
-    const m = moduleSplit.join('/');
-    if ({}.hasOwnProperty.call(alias, m)) {
-      aliasPath = alias[m];
-      break;
-    }
-    moduleSplit.pop();
-  }
-
-  // no alias mapping found
-  if (!aliasPath) {
-    return null;
-  }
-
-  // remove legacy "npm:" prefix for npm packages
-  aliasPath = aliasPath.replace(/^(npm:)/, '');
-  const newPath = sourcePath.replace(moduleSplit.join('/'), aliasPath);
-
-  // alias to npm module don't need relative mapping
-  if (aliasPath[0] !== '.') {
-    return newPath;
+function checkIfPackageExists(modulePath, currentFile) {
+  try {
+    resolve.sync(modulePath, {
+      basedir: currentFile,
+      extensions: nodeExtensions,
```

Comment:
I'm not sure about this one - the packages are 3rd party and so they will probably have standard extensions.

OTOH it shouldn't hurt much to include more files in the lookup, as long as they are package-ish (what should be ensured by `require.resolve`). In the worst case there will be false positives, which would result in some warnings being silenced, but it doesn't sound very bad.

I'll go with your suggested change.